hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -484,20 +484,28 @@ public Config setMapConfigs(Map<String, MapConfig> mapConfigs) {
     }
 
     /**
-     * Returns the cache config with the given name or {@code null} if there is none.
+     * Returns a read-only {@link CacheSimpleConfig} configuration for the given name.
+     * <p>
      * The name is matched by pattern to the configuration and by stripping the
      * partition ID qualifier from the given {@code name}.
+     * If there is no config found by the name, it will return the configuration
+     * with the name {@code default}.
      *
-     * @param name name of the cache config
-     * @return the cache configuration or {@code null} if none was found
+     * @param name name of the cardinality estimator config
+     * @return the cardinality estimator configuration
```

Comment:
Something went wrong with rebasing here? this javadoc should be about cache configs.